.github/workflows/build.yml

```diff
@@ -156,6 +156,8 @@ jobs:
       - name: Bindgen CLI
         run: cargo install bindgen-cli --locked
 
+      - uses: Swatinem/rust-cache@v2
```

Comment:
Is this necessary?

Reply:
hmmm - that was a mistake was poking around with caching rust builds and it shouldn't be in there

Reply:
removed